vcgencmd: add page (#6309)
tldr-pages/tldr: pages/linux/vcgencmd.md

# vcgencmd

> Print system information for a Raspberry Pi.
> More information: <https://www.raspberrypi.org/documentation/raspbian/applications/vcgencmd.md>.

- List all available commands:

`vcgencmd commands`

- Print the current CPU temperature:

`vcgencmd measure_temp`

> - Print the current voltage usage:

Author: ```suggestion
- Print the current voltage:
```

> `vcgencmd measure_volts`

Reviewer: According to the docs you have to specify a specific block here (`{{core|sdram_c|sdram_i|sdram_p}}`)

Author: % vcgencmd measure_volts
volt=1.2000V

Reviewer: Then are those maybe optional? Should  we include them?

Also, I'm not really happy with `voltage usage`, since that's technically not really correct. I'd just say `voltage`.

Author: Yes, voltage usage is incorrect

> - Print whether the CPU has throttled:

Reviewer: I think this is more accurate, since it doesn't return true or false.
```suggestion
- Print the throttled state of the system as a bit pattern:
```

`vcgencmd get_throttled`

> - Print the bootloader config (only available on Raspberry Pi 4 models):

Reviewer: I don't think there's a version 5 yet, but when there is, this page won't be outdated this way.
```suggestion
- Print the bootloader config (only available on Raspberry Pi 4 and above):
```

Author: We don't know whether it will have a EEPROM bootloader so I don't know about this one
